frontend/screens/vaccinationScreens/addVaccinationRecord.js

```diff
@@ -0,0 +1,294 @@
+import React, { useContext, useEffect, useState } from "react";
+import {
+  StyleSheet,
+  Text,
+  View,
+  ScrollView,
+  TouchableOpacity,
+} from "react-native";
+import { MainContext } from "../../context/MainContext";
+import CustomHeader from "../../components/header/custom-header";
+import CustomInputBox from "../../components/inputBox/custom-inputBox";
+import CustomButton from "../../components/button/custom-button";
+import DateTimePickerModal from "react-native-modal-datetime-picker";
+import IconBadge from "../../components/iconBadge/custom-iconBadge";
+import Toast from "react-native-toast-message";
+
+const AddUpdateVaccinationRecords = ({ update, recordId }) => {
+  const token =
+    "eyJhbGciOiJSUzI1NiIsImtpZCI6IjM2NGU4NTQ1NzI5OWQ5NzIxYjczNDQyZGNiNTQ3Y2U2ZDk4NGRmNTkiLCJ0eXAiOiJKV1QifQ.eyJpc3MiOiJodHRwczovL3NlY3VyZXRva2VuLmdvb2dsZS5jb20vbXktaGVhbHRoLXBhbC1kZjQxYSIsImF1ZCI6Im15LWhlYWx0aC1wYWwtZGY0MWEiLCJhdXRoX3RpbWUiOjE2MjkwNzQ5NzEsInVzZXJfaWQiOiJ4UzZNTXV5WENDWTVlR05tS2RvZnRrY3B6MjMzIiwic3ViIjoieFM2TU11eVhDQ1k1ZUdObUtkb2Z0a2NwejIzMyIsImlhdCI6MTYyOTA3NDk3MSwiZXhwIjoxNjI5MDc4NTcxLCJlbWFpbCI6InJpc2hhbnJhdG5hQGhvdG1haWwuY29tIiwiZW1haWxfdmVyaWZpZWQiOnRydWUsImZpcmViYXNlIjp7ImlkZW50aXRpZXMiOnsiZW1haWwiOlsicmlzaGFucmF0bmFAaG90bWFpbC5jb20iXX0sInNpZ25faW5fcHJvdmlkZXIiOiJwYXNzd29yZCJ9fQ.FSUq__5dhukERV3Zq_TdiKdqHRYNavsfibbOUL7RnL0x2jDhiPds8pARGQFaItwDpHR7Iet-UWztRoP0lZMSO_roVukYBWutC_lZdKM1a1mKzDs0Q3kV5bm0A5kZ7iT992AsMMM38EyPNoLom15wRa1oLH6MmAHd9LY6vpVSJj_FdxvWij0rlmYZB6_sF0DbMT2no6UgmYwwRkTzIg9A6YRFuH-z2MbAfIF8R11L_0zPTaB7Ubm9A3JW7R5_1GkQp8cCJu5HoHyXJAgRqMk-QqY12vF6e-YQvFoKRH41_MWyYhyenRhNXyrHe_HajvY3EC-PRcaynL-YfIkdmQKvVw";
```

Comment:
Remove the hard-coded token and perform a getToken() when sending the token to the api fetch to get the token from the SecureStore

Reply:
Agreed, good catch!

Reply:
Agreed, good catch @RishanR